config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openstack-k8s-operators/dataplane-operator
+  newName: quay.io/bshephar/dataplane-operator
```

Comment:
Oops

Reply:
Interesting, I wonder what changed this.